### PR TITLE
HimanshuSinghNegi LinkedFree profile added

### PR DESCRIPTION
**Pull request by HimanshuSinghNegi**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots
Screenshot (21)
Screenshot (22)

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2006"></a>

### CODE REVIEW

File: public/data/HimanshuSinghNegi.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/himanshusinghnegi",
+      "icon": "linkedin"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Open Souce Contribution",
+      "date": "October,14 2022",
```
Comment (Panquesito7) [inline public/data/HimanshuSinghNegi.json:26]: You might want to fix this minor issue here.
```suggestion
      "date": "October 14, 2022",
```

**Comment by HimanshuSinghNegi** [comment]
please review this change

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by imran1509 — COMMENTED** [review]
Looks good to me.
Welcome

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good to me! 👍

**Pull request merged**